.github/workflows/python-package.yml

```diff
@@ -0,0 +1,76 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+name: Python package
+on:
+  [ push, pull_request ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
+      matrix:
+        python-version: [ 3.8, 3.7, 3.6 ]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Cache conda
+      id: cache-miniconda
+      uses: actions/cache@v1
+      env:
+        CACHE_NUMBER: 0
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This is not set in the repo.

Reply:
It is necessary for using act locally. On GitHub actions it's neutral